File: tests/workflow/consumer/rust/src/consumer/broadcasts/groupb_groupc_structb.rs
use super::{protocol, Consumer, Context};
use clibri::client;

pub async fn handler<E: client::Error>(
    event: protocol::GroupB::GroupC::StructB,
    context: &mut Context,
    consumer: Consumer<E>,
) {
    context.broadcast.groupb_groupc_structb += 1;
    context.broadcast.check();
}

File: experimental/workflow_mt/producer/typescript/src/producer/responses/groupb.groupc.structb.ts
import { Response } from "../implementation/responses/groupb.groupc.structb";
import {
    Context,
    Producer,
    Identification,
    Filter,
    Protocol,
} from "../implementation/responses";

export function response(
    request: Protocol.GroupB.GroupC.StructB,
    consumer: Identification,
    filter: Filter,
    context: Context,
    producer: Producer
): Promise<Response> {
    return Promise.reject(
    	new Error(`Handler for Protocol.GroupB.GroupC.StructB isn't implemented.`)
    );
}